modules/ebi-metagenomics/eggnogmapper/main.nf

```diff
@@ -25,6 +25,7 @@ process EGGNOGMAPPER {
     def prefix = task.ext.prefix ?: "${meta.id}"
     def is_compressed = fasta.name.endsWith(".gz")
     def fasta_name = fasta.name.replace(".gz", "")
+    def dbmem = task.memory.toMega() > 40000 ? '--dbmem' : ''
```

Comment:
Should we have this parameter added explicitly instead of linking it with the memory requirements?. The DB alone is like 40G

Reply:
This is a failsafe for different profiles. For example, eggnog requires >40GB to load the db in memory, which my local profile for laptop cannot provide and therefore never finishes execution. Couldn't find a better place for this check

Reply:
Yeah, I understand but I'm not sure if "forcing" the db load when asking for loads memory is the best. I generaly think that is what our pipelines would like to do.
I think it's safe to leave it... we can change it if needed